### PR TITLE
Added Chainlink price aggregator w/ tests

### PR DESCRIPTION
**Pull request by yashpatel5400**
PR for changes requested in https://gitcoin.co/issue/Agoric/agoric-sdk/3200/100026006

refs #3200

### CODE REVIEW

**Comment by yashpatel5400** [comment]
Added a new test; fixed linting errors

**Review by michaelfig — COMMENTED** [review]
Most of my comments are stylistic, but there are a few logic changes buried in there.  Please take a look, and apply what you learn to the whole contract, not just the cases I pointed out.

The test code looks fine.  BTW, we have a lower bar for tests than for the contracts.

File: packages/zoe/src/contracts/priceAggregatorChainlink.js [hunk 1]
```diff
@@ -0,0 +1,743 @@
...
+          /* lastReportedRound = */ BigInt(0),
+          /* lastStartedRound = */ BigInt(0),
+          /* latestSubmission = */ BigInt(0),
+          /* index = */ oracleStatuses.keys().length,
+        );
+        oracleStatuses.init(oracleInstance, oracleStatus);
+      }
+      records.add(record);
+
+      const pushResult = async (_roundIdRaw, _submissionRaw) => {
```
[the remark this answers is not in the document]
Comment (yashpatel5400) [reply packages/zoe/src/contracts/priceAggregatorChainlink.js]: Oh interesting, yeah, that seems clean

**Comment by michaelfig** [comment]
Superseded by #3673

**Pull request closed without merging**